ReactAndroid/src/main/java/com/facebook/react/modules/network/ForwardingCookieHandler.java

```diff
@@ -184,8 +184,8 @@ protected void doInBackgroundGuarded(Void... params) {
         // We cannot catch MissingWebViewPackageException as it is in a private / system API
         // class. This validates the exception's message to ensure we are only handling this
         // specific exception.
-        // https://android.googlesource.com/platform/frameworks/base/+/master/core/java/android/webkit/WebViewFactory.java#98
-        if (message != null && message.contains("No WebView installed")) {
+        // https://android.googlesource.com/platform/frameworks/base/+/master/core/java/android/webkit/WebViewFactory.java#348
+        if (message != null && (exception.getClass().getCanonicalName().equals("android.webkit.WebViewFactory.MissingWebViewPackageException")) {
```

Comment:
👍

Reply:
Did you actually check this code and whether it works? The code here has a syntax error.

Reply:
@cpojer Sorry for the syntax error. I had actually tested this code in our internal fork. Made this mistake while committing here. Fixing this.